File: sunflower-core/src/main/java/org/tdf/sunflower/sync/Proposal.java
package org.tdf.sunflower.sync;

import org.tdf.sunflower.types.Block;

public class Proposal {
    private Block block;

    public Proposal(Block block) {
        this.block = block;
    }

    public Proposal() {
    }

    public Block getBlock() {
        return this.block;
    }

    public void setBlock(Block block) {
        this.block = block;
    }
}

File: sunflower-core/src/main/java/org/tdf/sunflower/sync/Proposal.kt
package org.tdf.sunflower.sync

import com.github.salpadding.rlpstream.annotation.RlpCreator
import com.github.salpadding.rlpstream.annotation.RlpProps
import org.tdf.sunflower.types.Block

@RlpProps("block")
data class Proposal @RlpCreator constructor(val block: Block)